crates/turborepo-lib/src/daemon/connector.rs

```diff
@@ -51,7 +51,7 @@ pub enum ForkError {
     Spawn(#[from] std::io::Error),
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Are we using the new clone functionality anywhere? I don't have an issue with the addition of the trait, just want to make sure I'm not missing anything.